Wikipedia/Code/WebViewController.m

```diff
@@ -133,6 +133,9 @@ - (void)viewDidLoad {
     [self displayArticle];
 }
 
+NSString* const WMF_UIScrollViewDidEndDeceleratingNotification = @"_UIScrollViewDidEndDeceleratingNotification";
```

Comment:
Need to observe both of these so we update both when TOC scrolls and when user scrolls the web view directly.